google-cloud-clients/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/models/RowMutationEntry.java

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.bigtable.data.v2.models;
+
+import com.google.api.core.InternalApi;
+import com.google.bigtable.v2.MutateRowsRequest;
+import com.google.protobuf.ByteString;
+import java.io.Serializable;
+import javax.annotation.Nonnull;
+
+/**
+ * Represents a list of mutations targeted at a single row. It's meant to be used as an parameter
+ * for {@link com.google.cloud.bigtable.data.v2.BigtableDataClient#newBulkMutationBatcher(String)}.
+ */
+public class RowMutationEntry implements MutationApi<RowMutationEntry>, Serializable {
+  private static final long serialVersionUID = 1974738836742298192L;
+
+  private final ByteString key;
+  private final Mutation mutation;
+
+  private RowMutationEntry(ByteString key, Mutation mutation) {
+    this.key = key;
+    this.mutation = mutation;
+  }
+
+  /** Creates a new instance of the mutation builder. */
+  public static RowMutationEntry create(@Nonnull String key) {
+    return create(ByteString.copyFromUtf8(key));
+  }
+
+  /** Creates a new instance of the mutation builder. */
+  public static RowMutationEntry create(@Nonnull ByteString key) {
+    return new RowMutationEntry(key, Mutation.create());
+  }
+
+  /** Creates a new instance of the mutation builder. */
+  public static RowMutationEntry create(@Nonnull String key, @Nonnull Mutation mutation) {
```

Comment:
Is it necessary to expose this?

Reply:
I don't think it is necessary, I copied these to have consistency between `RowMutation`

Reply:
I think those were added to RowMutation for explicit usage by the hbase client

Reply:
Thanks for clarifying it, I have removed those from `RowMutationEntry`.